pkgs/applications/editors/emacs/elisp-packages/manual-packages/lsp-bridge/default.nix

```diff
@@ -0,0 +1,124 @@
+{ lib
+, python3
+, trivialBuild
+, fetchFromGitHub
+, substituteAll
+, markdown-mode
+, posframe
+, yasnippet
+, git
+, go
+, gopls
+, pyright
+, tempel
+, runCommand
+, lsp-bridge
+, unstableGitUpdater
+}:
+
+let
+  python = python3.withPackages (ps: with ps; [
+    epc
+    orjson
+    sexpdata
+    six
+  ]);
+in
+trivialBuild {
+  pname = "lsp-bridge";
+  version = "unstable-2023-03-06";
+
+  src = fetchFromGitHub {
+    owner = "manateelazycat";
+    repo = "lsp-bridge";
+    rev = "b8dcb59790f390767fb5ceb2f8b11c6938456738";
+    sha256 = "sha256-6KyvRwwIMVolc8ufl0sqMR35d2MLLDpeDf86690LP+U=";
+  };
+
+  patches = [
+    # Hardcode the python dependencies needed for lsp-bridge, so users
+    # don't have to modify their global environment
+    (substituteAll {
+      src = ./hardcode-python-dependencies.patch;
+      inherit python;
+    })
+  ];
+
+  packageRequires = [
+    markdown-mode
+    posframe
+    yasnippet
+  ];
```

Comment:
Could you also add [acm-terminal][1] for terminal support?

Just a suggestion.

[1]: https://github.com/twlz0ne/acm-terminal

Reply:
I think either adding acm-terminal in another commit of this pr or in its own pr is fine.